lib/email_checker/loader.ex

```diff
@@ -1,16 +1,37 @@
 defmodule EmailChecker.Loader do
   use Application
 
-  def default_dns do
-    Application.get_env(:email_checker, :default_dns) || {4, 2, 2, 1}
+  defp default_dns do
+    Application.get_env(:email_checker, :default_dns, :system)
+  end
+
+  defp also_dns do
+    Application.get_env(:email_checker, :also_dns, [])
+  end
+
+  defp insert_ns(ns = {w,x,y,z})
+    when is_integer(w) and is_integer(x) and is_integer(y) and is_integer(z)
+  do
+    # HACK: add a default and "also" DNS when `inet_db` is not yet available
+    # http://stackoverflow.com/questions/33811899/elixir-errors-using-inet-res-library?noredirect=1#comment55418047_33811899
+
+    # Insert the specified dns server at the head of the nameserver list
```

Comment:
Do you think we need this comment? Because it just explains what the function does.